website/templates/project/nodes_privacy.mako

```diff
@@ -0,0 +1,111 @@
+<div id="nodesPrivacy" class="modal fade" div style="display: none;">
+    <div class="modal-dialog modal-md">
+        <div style="display: none;" data-bind="visible: true">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" data-bind="click: clear" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h3 class="modal-title" data-bind="text:pageTitle"></h3>
+            </div>
+
+            <div class="modal-body">
+
+                <!-- warning page -->
+
+                <div data-bind="if: page() == 'warning'">
```

Comment:
Minor: It would be good to used named constants here:

```html
<div data-bind="if: page() == WARNING">
```

This prevents programmer error, since a misspelling of the constant will result in an error.

This will require you to define the constants on the view model:

```js
self.WARNING = 'warning';
```